### ISSUE TITLE
please make deploy to heroku button

### ISSUE THREAD

**Issue by raisedhand** [kotborealis/nicovideo-yt-reuploader#1]
hi kotborealis ,
please if it's possible make deploy to heroku button for free of monthly vps charge.
waiting for your response
finest regards

**Comment by kotborealis** [comment]
It's a CLI-only application, I don't think it's suitable for heroku.
Maybe I'm missing something and you can now deploy CLI-apps to heroku? 🤔

**Comment by raisedhand** [comment]
yeah you can do it , just make worker for your important command must be awake all the time in Procfile named in your root of app. heroku have wide range of buildpacks can be found here :
https://elements.heroku.com/buildpacks
also you can app.json or heroku.yml to requirement env values must be set before deploy for automate all the things.
